themes/spirit/topics/water.lua

```diff
@@ -49,13 +49,14 @@ themepark:add_proc('area', function(object, data)
 end)
 
 themepark:add_proc('way', function(object, data)
+    local name = object.tags.name
```

Comment:
This should appear within the then block. There's no need to get the name if the object isn't going to be used. It also doesn't need a local variable since it's only used the once.